nilearn/plotting/html_stat_map.py

```diff
@@ -0,0 +1,97 @@
+import os
```

Comment:
As always, it would be good to have a module-level docstring to describe what this file is about.